ddtrace/bootstrap/sitecustomize.py

```diff
@@ -54,29 +54,15 @@ def update_patched_modules():
     modules_to_patch = os.environ.get("DATADOG_PATCH_MODULES")
     if not modules_to_patch:
         return
-    for patch in modules_to_patch.split(","):
-        if len(patch.split(":")) != 2:
-            log.debug("skipping malformed patch instruction")
-            continue
 
-        module, should_patch = patch.split(":")
-        if should_patch.lower() not in ["true", "false"]:
+    modules = parse_tags_str(modules_to_patch)
+    for module, should_patch in modules.items():
+        should_patch = should_patch.lower()
+        if should_patch not in ["true", "false"]:
```

Comment:
`ddtrace.utils.formats.asbool` ?

Reply:
yeah we may as well do it here too. It wasn't originally an `asbool` but I see no reason to not use it here. 😄